From f9485d1bfad9386f802e5b1411ed9fd086f76e22 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 21 Dec 2017 13:40:18 +0900
Subject: [PATCH 674/909] arm64: dts: r8a7795-es1-h3ulcb: Add MMNGRBUF driver
 node

commit 557a6af611ef1603dece80672b2c029765fd31b2 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds MMNGRBUF external kernel module for H3ULCB board
on R8A7795 ES1.x SoC.

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
index 7b5c6915d92f..e259922c6f66 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1-h3ulcb.dts
@@ -66,6 +66,10 @@
 		compatible = "renesas,mmngr";
 		memory-region = <&mmp_reserved>, <&lossy_decompress>;
 	};
+
+	mmngrbuf {
+		compatible = "renesas,mmngrbuf";
+	};
 };
 
 &sdhi2 {
-- 
2.17.1

